### ISSUE TITLE
sendMp4AsSticker() not responding. 

### ISSUE THREAD

**Issue by TVARecordings** [open-wa/wa-automate-nodejs#3216]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Business account

### Mode

EASY API/CLI

### Current Behavior

Hey SHAH. 

Unfortunately, sendMp4AsSticker() has been broken.
I bet it's again the case that WA has changed something.

Just letting u know.

PS: When using sendMp4AsSticker(), it will return 'true' instantly, but the sticker is not send. 

### Expected Behavior

Expected it to send the sticker :-)

### Steps To Reproduce

sendMp4AsSticker()

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.3000.1014085685",
  "WA_AUTOMATE_VERSION": "4.71.12",
  "BROWSER_VERSION": "Chrome/125.0.6422.142",
  "START_TS": 1717957835458,
  "RAM_INFO": "Total: 68.62 GB | Free: 44.65 GB",
  "PPTR_VERSION": "19.11.1",
  "LATEST_VERSION": true,
  "CLI": false
}
```

### Environment

```markdown
- OS: Windows 10
- Node: 20.11.1
- npm: 10.2.4
```

### Anything else?

Keep up the good work :)

**Comment by aymenitc** [comment]
Please Shah, waiting for a fix kindly

**Comment by TVARecordings** [comment]
Bump

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.71.14](https://github.com/open-wa/wa-automate-nodejs/compare/4.71.13...4.71.14) (2024-06-26)

- Release 4.71.14 [`e092bbf`](https://github.com/open-wa/wa-automate-nodejs/commit/e092bbf650fc61a57487d9b621ffa7b1f4498578)
- 🐛 fix `sendMp4AsSticker` #3216 [`e059f60`](https://github.com/open-wa/wa-automate-nodejs/commit/e059f6026574279d701415624a8d834d33e338af)
- updated types-only package [`31fe831`](https://github.com/open-wa/wa-automate-nodejs/commit/31fe83140f9c8a86dce109c4181debe5a66fbcf0)
